config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    // Name of the container file excluding the file extension (e.g. "name" yields "name.js" when published)
+    containerName: 'mojito',
+    // Development S3 container
+    s3BucketDev: 'mojito-js/jsdev',
+    // Production S3 container
+    s3BucketPRD: 'mojito-js/js',
+    // S3 Region
+    s3Region: 'ap-southeast-2'
+};
```

Comment:
Do we need to show an example of `config.allowMultiInstance` setting in this file and in the example file too?

Reply:
@kingo55 
added.